tests/integration/api_v3/http/tokens.test.js

```diff
@@ -28,7 +28,7 @@ const {
 
 const baseUrl = config.SERVICE_ENDPOINT;
 const baseUrlV3 = `${baseUrl}/api/v3`;
-const endpoint = `${baseUrlV3}/tokens`;
+const endpoint = `${baseUrlV3}/token`;
```

Comment:
Also, please rename the file accordingly. And, kindly do not forget to make the corresponding changes in the RPC tests.
```suggestion
const endpoint = `${baseUrlV3}/token/balances`;
```